src/components/CommentsSection.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import Comment from './Comment';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+import Button from './Button';
+import './CommentsSection.css';
+
+const CommentsSection = () => {
+    const [comments, setComments] = useState([
+        { commentId: 1, author: "Jakob_Miller", text: "Feel free to ask me questions!" },
+        { commentId: 2, author: "Emily_Hartman", text: "Is there a specific age or breed requirement for the dogs?" },
+        { commentId: 3, author: "Jakob_Miller", text: "Nope, all breeds and ages are welcome to participate!" }
+    ]);
+    const [newComment, setNewComment] = useState("");
+
+    // Function to handle sending a new comment
+    const handleSend = () => {
+        if (newComment.trim()) {
+            const newCommentObj = {
+                commentId: comments.length + 1,
+                author: "Current_User",
+                text: newComment
+            };
+            setComments([...comments, newCommentObj]);
+            setNewComment("");
+        }
+    };
+
+    // Function to handle deleting a comment
+    const handleDelete = (commentId) => {
+        const confirmed = window.confirm("Are you sure you want to delete this comment?");
```

Comment:
window.confirm is a blocking call, meaning it halts the execution of JavaScript until the user responds. This can lead to a poor user experience and accessibility concerns. I'd recommend that you make a custom dialog/modal/toast instead.